### PR TITLE
DOC-602: Update documentation for the Tasking class

### PR DESCRIPTION
**Pull request by dustgalactic**
* Reference
* Tutorials
* Changelog
* Announcement bar

### CODE REVIEW

**Review by andher1802 — APPROVED** [review]
LGTM! Nice work. Thanks @dustgalactic

File: docs/reference/tasking-reference.md [hunk 1]
```diff
@@ -1,7 +1,227 @@
...
+| ------------------- | ----------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data_product_id`   | `str`                                                                         | The data product ID.                                                                                                                          |
+| `name`              | `str`                                                                         | The tasking order name.                                                                                                                       |
+| `acquisition_start` | `Union[str, datetime]`                                                        | The start date of the acquisition period in the `YYYY_MM_DD` format.                                                                          |
```
Comment (seedlit) [inline docs/reference/tasking-reference.md:31]: @dustgalactic the format is incorrect.
It should be "YYYY-MM-DD".

Please see [this](https://github.com/up42/up42-py/blob/f0005fd569de57d43746acf4dbadcffbb18d2ae3/up42/tasking.py#L57) for reference.

Same for acquisition_end.

File: docs/reference/tasking-reference.md [hunk 2]
```diff
@@ -1,7 +1,227 @@
...
+| Type   | Description                    |
+| ------ | ------------------------------ |
+| `JSON` | A list of feasibility studies. |
+
+<h3> Example </h3>
+
+```python
+tasking.get_feasibility(
+    feasibility_id=None,
+    workspace_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
```
Comment (seedlit) [inline docs/reference/tasking-reference.md:104]: this is a fake id right?

File: docs/reference/tasking-reference.md [hunk 3]
```diff
@@ -1,7 +1,227 @@
...
+| Type   | Description                                   |
+| ------ | --------------------------------------------- |
+| `dict` | Feasibility option confirmation and metadata. |
+
+<h3> Example </h3>
+
+```python
+tasking.choose_feasibility(
+    feasibility_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
+    accepted_option_id="a0d443a2-41e8-4995-8b54-a5cc4c448227"
```
Comment (seedlit) [inline docs/reference/tasking-reference.md:143]: generic question: all the ids are fake right?

File: docs/tasking.md [hunk 4]
```diff
@@ -107,29 +107,44 @@ order
...
 
 After an order is placed, the Operations team will look at the order feasibility. They will evaluate the tasking parameters with the provider, and then will present the following assessments:
 
 - The order is possible with the given parameters.
 - The order requires modifications with suitable options proposed.
 
-You will receive an email notifying you when your order feasibility has been assessed. To proceed, follow these steps:
+You will receive an email notifying you when your order feasibility has been assessed. Using your order ID, review the proposed feasibility options as follows:
+```python
+tasking.get_feasibility(order_id="68567134-27ad-7bd7-4b65-d61adb11fc78")
```
Comment (seedlit) [inline docs/tasking.md:119]: Nice!

File: docs/tasking.md [hunk 5]
```diff
@@ -107,29 +107,44 @@ order
...
+    feasibility_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
+    accepted_option_id="a0d443a2-41e8-4995-8b54-a5cc4c448227"
+)
+```
 
-1. Open the [console](https://console.up42.com) and go to **Tasking** → **My orders**.
-2. Select your order and click **Feasibility**.
-3. Choose a suitable option. If none of the options are suitable, [contact](https://up42.com/company/contact-ordering) the Operations team.
-4. Click **Confirm** chosen option.
+If none of the options are suitable, [contact](https://up42.com/company/contact-ordering) the Operations team.
```
Comment (seedlit) [inline docs/tasking.md:130]: wasn't even aware this existed!

File: docs/reference/tasking-reference.md (hunk 1 above)
Comment (dustgalactic) [reply docs/reference/tasking-reference.md:31]: Thank you for catching that. I'll fix in a new PR

File: docs/reference/tasking-reference.md (hunk 2 above)
Comment (dustgalactic) [reply docs/reference/tasking-reference.md:104]: We have a couple of sample IDs that we use in the API reference, it's them

File: docs/reference/tasking-reference.md (hunk 1 above)
Comment (dustgalactic) [reply docs/reference/tasking-reference.md:31]: https://up42.atlassian.net/browse/DOC-616

**Pull request merged**